### ISSUE TITLE
sr_audit does not start for non-admin users.

### ISSUE THREAD

**Issue by petersilva** [MetPX/sarracenia#22]
sr_audit used to be only for administrative users.  But in later versions in 2018 ( > 2.18.03xxxx )
We have added that it looks for dead logs to figure out if user processes have died.  With the heartbeat processing, there should always be recent events in the log.

it doesn´t start up for non-administrative users...

```
peter@idefix:~$ sr_audit foreground
2018-05-29 11:40:52,757 [INFO] sr_audit has no admin set...
peter@idefix:~$ 
```

**Comment by petersilva** [comment]
Fixed in git.  now it starts... Want to make sure it does something useful also though.

**Comment by petersilva** [comment]
released in 2.18.06b2
